smart_contracts/contract_as/assembly/dictionary.ts

```diff
@@ -73,3 +73,26 @@ export function dictionaryPut(uref: URef, key: String, value: CLValue): void {
         valueBytes.length
     );
 }
+
+/**
+ * Reads `value` under `dictionary-key` in a dictionary.
+ * @category Storage
+ */
+export function dictionaryRead(dictionaryKey: Key): Uint8Array | null {
```

Comment:
move next to dictionaryGet above

modify the comment on line 39 above to be very clear that the key argument Get expects is dictionary_item_key
>* Reads the value under the dictionary item `key`.

Reply:
Moved, and the comment for `Get` has been updated in https://github.com/casper-network/casper-node/pull/3161/commits/862d2b060f8391957da23d76025c2b8a6fd92c5f